### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by geolffreym**

### CODE REVIEW

**Comment by geolffreym** [comment]
on it!! @alanshaw

**Comment by geolffreym** [comment]
ready @alanshaw check please!!

File: README.md [hunk 1]
```diff
@@ -102,6 +102,8 @@ Install dependencies:
 npm install orbit-db ipfs
```
Comment (aphelionz) [inline README.md:102]:
```suggestion
npm install orbit-db ipfs@~0.46.0
```

File: README.md [hunk 2]
```diff
@@ -102,6 +102,8 @@ Install dependencies:
...
 ```
 
+***NOTE!*** Compatible with **js-ipfs versions <= 0.46**
```
Comment (aphelionz) [inline README.md:105]: Sorry! I should have been more clear. I liked where this was before but also commit my suggestion as well :)

**Comment by alanshaw** [comment]
Not sure why I’m being tagged. What needs to happen for upgrade to new js-ipfs?

**Comment by aphelionz** [comment]
I wasn't sure either. I just need to spend some time with it and OrbitDB to see where the issues are.

File: README.md (hunk 2 above)
Comment (geolffreym) [reply README.md:105]: oh right!! on it

**Pull request merged**